kubespawner/templates/form.html

```diff
@@ -18,7 +18,7 @@ <h3>{{ profile.display_name }}</h3>
             {%- for k, option in profile.profile_options.items() %}
               <div class="option">
                 <label for="profile-option-{{ profile.slug }}-{{ k }}">{{ option.display_name }}</label>
-                <select name="profile-option-{{ profile.slug }}-{{ k }}" class="form-control">
+                <select name="profile-option-{{ profile.slug }}-{{ k }}" class="js-profile-option-select form-control">
```

Comment:
Line 1: Update `for` attribute of label to point to the radio button rather than the select item
Line 2-3: Autoformatting to split lines, and consistently putting js-x classes last.

```suggestion
                <label for="profile-item-{{ profile.slug }}">{{ option.display_name }}</label>
                <select name="profile-option-{{ profile.slug }}-{{ k }}"
                        class="form-control js-profile-option-select">
```

Reply:
@consideRatio did editing the `for` attribute work for you? It didn't seem to work for me when I tried, but I can try again / maybe I messed something up.

Reply:
Yepp, this is making the `for` change from the main branch, and testing on chrome of the latest/very recent version from ubuntu 22.04

![change](https://github.com/jupyterhub/kubespawner/assets/3837114/00c89bf2-4b00-47f9-a398-c2bbfdfe6d9c)